cmd/manager/main.go

```diff
@@ -308,7 +112,7 @@ type Config struct {
 func parseConfiguration() Config {
 	var globalAPISecretName string
 	config := Config{}
-	flag.StringVar(&config.AtlasDomain, "atlas-domain", "https://cloud.mongodb.com/", "the Atlas URL domain name (with slash in the end).")
+	flag.StringVar(&config.AtlasDomain, "atlas-domain", operator.DefaultAtlasDomain, "the Atlas URL domain name (with slash in the end).")
```

Comment:
> "(with slash in the end)."

Unrelated for this PR but should we make it so that this comment is not needed and the `/` is auto auto appended as needed?

Reply:
Why is this a problem?

Reply:
Maybe not important, but if you forget the `/` in the helm chart values you will not have this nice help message available to you there. I mean, usually the less explanations needed the better.
But is off topic, not for this PR.